### PR TITLE
August23 update

### LINKED ISSUE #291: Godot
https://godotengine.org/download/android/

### PR DESCRIPTION
**Pull request by shuvashish76**
Closes #291 , #290
New category added - Battery, Research Development, Sandboxing

### CODE REVIEW

**Comment by farline99** [comment]
[Sent my pull request just in time](https://github.com/offa/android-foss/pull/304) :D
Will delete the overlapping applications now.

**Comment by shuvashish76** [comment]
Oopse, sorry for my duplicates & thanks for your PR :D

**Comment by farline99** [comment]
Done ❤️

File: README.md [hunk 1]
```diff
@@ -526,9 +543,16 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**AntennaPod**](https://github.com/AntennaPod/AntennaPod) <sup>**[[F-Droid](https://f-droid.org/app/de.danoeh.antennapod)]**</sup>
 * [**Audio Anchor**](https://github.com/flackbash/AudioAnchor) <sup>**[[F-Droid](https://f-droid.org/app/com.prangesoftwaresolutions.audioanchor)]**</sup>
 * [**Escapepod**](https://codeberg.org/y20k/escapepod) <sup>**[[F-Droid](https://f-droid.org/app/org.y20k.escapepod)]**</sup>
+* [**Podverse**](https://github.com/podverse/podverse-fdroid) <sup>**[[F-Droid](https://f-droid.org/app/com.podverse.fdroid)]**</sup>
 * [**Tsacdop**](https://github.com/stonega/tsacdop) <sup>**[[F-Droid](https://f-droid.org/app/com.stonegate.tsacdop)]**</sup>
 * [**Voice**](https://github.com/PaulWoitaschek/Voice) <sup>**[[F-Droid](https://f-droid.org/app/de.ph1b.audiobook)]**</sup>
 
+### • Research & Development
+
+* [**Godot Editor**](https://github.com/godotengine/godot/) <sup>**[[F-Droid](https://f-droid.org/app/org.godotengine.editor.v3)]**</sup>
```
Comment (offa) [inline README.md:552]:
```suggestion
* [**Godot Editor**](https://github.com/godotengine/godot) <sup>**[[F-Droid](https://f-droid.org/app/org.godotengine.editor.v3)]**</sup>
```

No really relevant though :smile:.

File: README.md [hunk 2]
```diff
@@ -465,6 +481,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Phonograph Plus**](https://github.com/chr56/Phonograph_Plus) <sup>**[[F-Droid](https://f-droid.org/app/player.phonograph.plus)]**</sup>
 * [**raag**](https://github.com/raag-music/raag) <sup>**[[F-Droid](https://f-droid.org/app/in.amfoss.raag)]**</sup>
 * [**Simple Music Player**](https://github.com/SimpleMobileTools/Simple-Music-Player) <sup>**[[F-Droid](https://f-droid.org/app/com.simplemobiletools.musicplayer)]**</sup>
+* [**SimpMusic**](https://github.com/maxrave-dev/SimpMusic) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.maxrave.simpmusic)]**</sup>
```
Comment (offa) [inline README.md:484]: It's on F-Droid already: https://f-droid.org/app/com.maxrave.simpmusic

According to it's readme it's still beta though. Maybe we put it on hold, but no strong opinion here – no objection.

File: README.md [hunk 3]
```diff
@@ -136,9 +136,17 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Catima**](https://github.com/CatimaLoyalty/Android) <sup>**[[F-Droid](https://f-droid.org/app/me.hackerchick.catima)]**</sup>
 * [**QR & Barcode Scanner**](https://github.com/wewewe718/QrAndBarcodeScanner) <sup>**[[F-Droid](https://f-droid.org/app/com.example.barcodescanner)]**</sup>
 
+### • Battery
+
+* [**ACCA**](https://github.com/MatteCarra/AccA) <sup>**[F-Droid](https://f-droid.org/app/mattecarra.accapp)]**</sup>
+* [**Battery Tool**](https://github.com/Domi04151309/BatteryTool) <sup>**[[F-Droid](https://f-droid.org/app/io.github.domi04151309.batterytool)]**</sup>
+* [**BatteryBot Pro**](https://github.com/darshan-/Battery-Indicator-Pro) <sup>**[[F-Droid](https://f-droid.org/app/com.darshancomputing.BatteryIndicatorPro)]**</sup>
```
Comment (offa) [inline README.md:143]: Is this project still active?

**Comment by offa** [comment]
Thanks @shuvashish76! I have added some minor comments, but only editorial.

Is there any order necessary to merge your PRs @farline99 @shuvashish76?

**Comment by farline99** [comment]
> Is there any order necessary to merge your PRs @farline99 @shuvashish76?
No, PRs don't overlap, so it doesn't matter.

File: README.md [hunk 4]
```diff
@@ -692,14 +722,15 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
...
+* [**WallMan**](https://gitlab.com/colorata/wallman) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/com.colorata.wallman)]**</sup>
+* [**Wall You**](https://github.com/Bnyro/WallYou) <sup>**[[F-Droid](https://f-droid.org/app/com.bnyro.wallpaper)]**</sup>
 * [**Zade's Wallpaper**](https://github.com/zadeviggers/wallpaper) <sup>**[[F-Droid](https://www.f-droid.org/app/net.viggers.zade.wallpaper)]**</sup>
 
 ### • Weather
 
+* [**Breezy Weather**](https://github.com/breezy-weather/breezy-weather) <sup>**[[IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/org.breezyweather)]**</sup>
 * [**Clima**](https://codeberg.org/Lacerte/clima) <sup>**[[F-Droid](https://www.f-droid.org/app/co.prestosole.clima)]**</sup>
 * [**Forecastie**](https://github.com/martykan/forecastie) <sup>**[[F-Droid](https://f-droid.org/app/cz.martykan.forecastie)]**</sup>
-* [**Geometric Weather**](https://github.com/WangDaYeeeeee/GeometricWeather) <sup>**[[F-Droid](https://f-droid.org/app/wangdaye.com.geometricweather)]**</sup>
```
[the remark this answers is not in the document]
Comment (farline99) [reply README.md:702]: Breezy Weather (a fork) replaced Geometric, so it's being removed.

File: README.md (hunk 1 above)
Comment (farline99) [reply README.md:552]: Perfectionism :D:D:D

File: README.md (hunk 2 above)
Comment (shuvashish76) [reply README.md]: I guess they're really close for stable release hence published on F-Droid ⸮ Updated F-Droid link.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (shuvashish76) [reply README.md:702]: https://github.com/WangDaYeeeeee/GeometricWeather/issues/492

[the remark this answers is not in the document]
Comment (shuvashish76) [reply README.md:702]: Thanks @farline99

File: README.md (hunk 3 above)
Comment (shuvashish76) [reply README.md:143]: See my comment [here](https://github.com/MuntashirAkon/android-debloat-list/pull/29#discussion_r1303326164).

**Comment by shuvashish76** [comment]
R&D : This new* category created to keep track of useful/active apps similar to NetHunter store, which is seems abandoned. Only they actively develop NetHunter & maintain NetHunter Terminal

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (offa) [reply README.md:702]: Thanks for the info @farline99

File: README.md [hunk 5]
```diff
@@ -262,6 +262,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • Drawing
 
+* [**FreePaint**](https://github.com/pastthepixels/FreePaint) <sup>**[[F-Droid](https://www.f-droid.org/app/io.github.pastthepixels.freepaint)]**</sup>
```
Comment (offa) [inline README.md:266]: The link ends in a 404. The search lists it already, so I guess it'll be reachable soon.

Comment (offa) [reply README.md:266]: Link active now

**Comment by farline99** [comment]
How it is not merged yet?)

**Comment by offa** [comment]
It has merge conflicts. 

@shuvashish76 let me know if I can help here :-)

**Comment by shuvashish76** [comment]
Closing it then, I'll add them next time, thanks.

**Pull request closed without merging**